lfs/util.go

```diff
@@ -326,3 +327,39 @@ func FileExistsOfSize(path string, sz int64) bool {
 
 	return !fi.IsDir() && fi.Size() == sz
 }
+
+func CopyFileContents(src string, dst string) error {
+	tmp, err := ioutil.TempFile(TempDir, filepath.Base(dst))
+	if err != nil {
+		return err
+	}
+	defer func() {
+		tmp.Close()
+		_ = os.Remove(tmp.Name())
```

Comment:
Not worth changing unless you have to make another change, but I tend to omit the `_ =` if I'm not going to do anything with it. Just some microfeedback :)